src/app/shared/snackbar-options.ts

```diff
@@ -0,0 +1,21 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+export const defaultSnackbarOptions = { duration: 5000, panelClass: 'center' };
```

Comment:
If you're going to add this now, can you search and replace all relevant cases of snackbar then so we don't forget?

Reply:
Done